README.adoc

```diff
@@ -2,13 +2,23 @@
 :experimental:
 ifndef::env-github[:toc:]
 
+ifdef::env-github[]
```

Comment:
When adding header attributes, there should be no empty lines between them. However, I don't think we need admonitions in this README, so we can drop this whole ifdef anyway.